### PR TITLE
Added Mongo and updated packages

### PR DESCRIPTION
**Pull request by Sp0Q1**
Also fixed package update issues

### CODE REVIEW

**Comment by maapteh** [comment]
@naugtur is there any chance you can review this? I know the project is old, but with this change its more usable.

**Comment by naugtur** [comment]
Will review. Can't promise to do it by the end of September. But will try

File: package.json [hunk 1]
```diff
@@ -42,10 +42,12 @@
   "dependencies": {
```
Comment (maapteh) [inline package.json]: don't forget to update the engine version, everything below 10.16.3 is not advised so >= 10.16.3

**Review by maapteh — COMMENTED** [review]
Update engine in package.json >= 10.16.3 everything below is unsecure as we know off

**Comment by maapteh** [comment]
Thanks Naugtur! We will run the fork in our own environment and if we pick up things we wil update this PR. At least Frank :)

File: package.json (hunk 1 above)
Comment (SpenQ) [reply package.json]: done

**Comment by naugtur** [comment]
I'm seeing one potentially unhandled rejection there after switching to promises, but it's not like it was handled before... so we're good.

**Comment by maapteh** [comment]
Ok thanks for your review! Maybe we will look at it later more closely because we dont need lodash for server part can be plain js using array includes etc etc.

**Pull request merged**